### PR TITLE
Quality Profile Sync Updates

### PR DESCRIPTION
**Pull request by yammes08**
- Added quality profile sync configurations to main guide, SQP and anime config templates
- Added more comments with file information and links to relevant documentation pages
- Applied consistent naming to SQP profiles
- Removed duplicate `sqp-1.yml`

### CODE REVIEW

**Comment by yammes08** [comment]
@rcdailey - I wouldn't mind you casting your eyes over these updates if that's ok!

As well as the QP updates themselves, I'd appreciate your thoughts on the extra comments and header. My feeling was that it directs people to the relevant documentation sections for those areas, as we quite often have support questions about them.

File: radarr/anime-radarr.yml [hunk 1]
```diff
@@ -1,16 +1,67 @@
-# Updated: 2023-08-01
+###################################################################################################
+# Recyclarr Configuration Template: Anime Radarr                                                  #
+# Updated: 2023-08-13                                                                             #
+# Documentation: https://recyclarr.dev                                                            #
+# Note: If you are using multiple profiles in a single instance, please read the following        #
+# documentation about file merging:                                                               #
+# https://recyclarr.dev/wiki/yaml/config-examples/#merge-single-instance                          #
```
Comment (rcdailey) [inline radarr/anime-radarr.yml:7]: This is an excellent addition!

File: radarr/anime-radarr.yml [hunk 2]
```diff
@@ -1,16 +1,67 @@
...
+###################################################################################################
+
+# Instance Definition
+# This section defines your instance type, name, and authentication
+# Documentation: https://recyclarr.dev/wiki/yaml/config-reference/#basic
 radarr:
-  anime-radarr:  # A custom name used to identify this particular instance.
```
Comment (rcdailey) [inline radarr/anime-radarr.yml:3]: Good idea removing this comment. Now that all of the names match the files, I think customizing them is less important.

File: radarr/sqp/sqp-1-2160p.yml [hunk 3]
```diff
@@ -1,16 +1,72 @@
-# Updated: 2023-08-02
+###################################################################################################
+# Recyclarr Configuration Template: SQP-1 (2160p)                                                 #
+# Updated: 2023-08-13                                                                             #
```
Comment (rcdailey) [inline radarr/sqp/sqp-1-2160p.yml:3]: Just a thought but I bet we could come up with a script that automatically adds this header *and* updates the date stamp for files that are modified in Git. If you are interested in that, I can try to come up with something. Shouldn't be too hard.

**Review by rcdailey — APPROVED** [review]
This looks really great. I didn't go over every single file in detail, because it seems like the changes are pretty consistent between files. Thanks for giving me the opportunity to take a look!

File: radarr/sqp/sqp-1-2160p.yml (hunk 3 above)
Comment (yammes08) [reply radarr/sqp/sqp-1-2160p.yml:3]: Cool - I'll resolve here and open an issue for us to talk it over. Can take this as a leisurely one to implement!

**Pull request merged**